src/schema/v2/me/__tests__/bidders.test.js

```diff
@@ -52,5 +52,32 @@ describe("Me", () => {
         }
       )
     })
+
+    it("calls the gravity endpoint with the `filter` param if included", () => {
+      const query = `
+        {
+          me {
+            bidders(active: true) {
+              internalID
+            }
+          }
+        }
+      `
+      const response = () =>
+        Promise.resolve([{ id: "Foo ID" }, { id: "Bar ID" }])
```

Comment:
I don't inherently know the shape of these objects, but would it make sense to mock one of these to be representative of a non-active state to ensure it's filtered out as expected?

Reply:
The filtering happens at the gravity layer so all we need to test here is that the argument is properly forwarded.